package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ulsan",
+  "version": "1.0.0",
+  "description": "This is the web application server for the game Mazing Friends.",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "serve ./dist",
+    "dev": "nodemon server/server.js",
+    "webpack-dev": "webpack --watch",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ehdwn1212/ulsan.git"
+  },
+  "author": "DJ Kim, Anukul Shenoy, and Savy Brandt",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ehdwn1212/ulsan/issues"
+  },
+  "homepage": "https://github.com/ehdwn1212/ulsan#readme",
+  "devDependencies": {
+    "babel": "^6.23.0",
```

Comment:
Something I learned recently is that it's good practice to not use ^ to have fixed dependency versions